src/test/functional/queryServices.test/savedItems.integration.ts

```diff
@@ -429,13 +429,12 @@ describe('getSavedItems', () => {
       {
         sortBy: 'ARCHIVED_AT',
         sortOrder: 'DESC',
-        expectedUrls: ['http://abc', 'http://ijk'],
+        expectedUrls: ['http://ijk', 'http://def'],
```

Comment:
this now correctly uses the itemId as the 'fallback' for stable sort on timestamp collisions